packages/react-reconciler/src/__tests__/ReactSuspenseEffectsSemantics-test.js

```diff
@@ -148,6 +148,12 @@ describe('ReactSuspenseEffectsSemantics', () => {
 
   function Text({children = null, text}) {
     Scheduler.log(`Text:${text} render`);
+    React.useInsertionEffect(() => {
```

Comment:
It made sense to me to add `useInsertionEffect` to all of those helper components that are used throughout this test suite. 

It increases the coverage and might surface some subtle bugs in the future. I think that this matches the spirit of this file because tests that are explicitly concerned with layout effects behavior are also tracking passive effects (since they rely on those helper components). So I didn't see a reason why insertion effects should be skipped from here.

If you don't like the change - let me know and I can easily remove that and only leave the targeted test case for the fixed issue.